composer.json

```diff
@@ -2,11 +2,13 @@
     "name": "mailgun/mailgun-php",
     "description": "The Mailgun SDK provides methods for all API functions.",
     "require": {
-        "guzzlehttp/guzzle": "~5.0"
+        "guzzlehttp/psr7": "~1.2",
```

Comment:
You are correct. We should not be dependent on Guzzle's implementation here. We should use a message factory.. I'll update in a minute.